.jvmopts-travis

```diff
@@ -2,3 +2,4 @@
 
 -Dfile.encoding=UTF8
 -Dsbt.color=always
+-Dakka.actor.testkit.typed.timefactor=3
```

Comment:
time dilation is the wrong way of solving flaky tests, default 3 seconds should be enough, and for special initialization tasks there should be an explicit longer timeout

Reply:
👍🏼 Looking at alternatives.